### PR TITLE
Add UHDBits provider

### PR DESCRIPTION
**Pull request by lansinghd**
### Description of what this fixes:

This UHDBits provider is based off of the PTP provider and uses the Gazelle JSON API for searches.  

Searches are based on the title, year, quality, and media, but the API doesn't return the full release name for each result so returning information about group name, 3D, etc isn't available for now.  

Otherwise it seems to work well - any and all feedback is appreciated!

### CODE REVIEW

File: couchpotato/core/media/_base/providers/torrent/uhdbits.py [hunk 1]
```diff
@@ -0,0 +1,259 @@
...
+        'login': 'https://uhdbits.org/login.php',
+        'login_check': 'https://uhdbits.org/inbox.php',
+        'api': 'https://uhdbits.org/ajax.php',
+        'detail': 'https://uhdbits.org/torrents.php'
+    }
+
+    login_errors = 0
+    http_time_between_calls = 2
+
+    def _search(self, media, quality, results):
```
Comment (RuudBurger) [inline couchpotato/core/media/_base/providers/torrent/uhdbits.py]: Please use the searchOnTitle function here, like https://github.com/CouchPotato/CouchPotatoServer/blob/master/couchpotato/core/media/_base/providers/nzb/omgwtfnzbs.py#L28
That function get's called multiple times with different title possibilities and does the work for you.

**Comment by RuudBurger** [comment]
Thanks! If you fix the _search to the _searchOnTitle, I can merge it.

**Comment by lansinghd** [comment]
Ah, I completely missed that function - thanks for the notes!  Fixed and tested, it successfully loops and tests multiple titles now.

**Comment by theCheek** [comment]
I just joined this tracker and its amazing, so would love for it to be on the master branch (my seedbox doesn't allow for the switch to develop). Any idea when this update will merge with master?

**Comment by rabidfurball** [comment]
me too!  Please merge this in.

**Pull request closed without merging**